rtcstats.js

```diff
@@ -30,6 +30,7 @@
   function trace() {
     //console.log.apply(console, arguments);
     // TODO: drop getStats when not connected?
+    arguments.push(new Date().getTime());
```

Comment:
@fippo `arguments` is not an array object. Please [avoid mutating it](https://github.com/petkaantonov/bluebird/wiki/Optimization-killers#3-managing-arguments)

```js
var args = Array.prototype.slice.call(arguments);
args.push(new Date().getTime());